src/gatsby-node.js

```diff
@@ -10,33 +10,128 @@ exports.sourceNodes = (
   delete configOptions.plugins
 
   const sources = []
+  const dummyData = {
+    id: 'dummy',
+    appointment: '',
+    bio: '',
+    building: '',
+    consult_service: '',
+    experience: '',
+    grant_contract: '',
+    honor_award: '',
+    innovate_enterpreneur: '',
+    patent_invention: '',
+    pf_email: '',
+    pf_work_fax: '',
+    pf_first_name: '',
+    pf_last_name: '',
+    pf_work_phone: '',
+    pf_title: '',
+    pf_username: '',
+    website: '',
+    research: '',
+    room: '',
+    photo_base64: '',
+    notable_courses: '',
+    school: '',
+    service_university: [
+      {
+        org: '',
+        member_type: '',
+      },
+    ],
+    service_professional: [
+      {
+        title: '',
+        org: '',
+      },
+    ],
+    education: [
+      {
+        dty_comp: '',
+        deg: '',
+        degother: '',
+        school: '',
+        state: '',
+        country: '',
+        major: '',
+      },
+    ],
+    member: [
+      {
+        org: '',
+        status: '',
+        orgabbr: ''
+      },
+    ],
+    intellcont: [
+      {
+        contype: '',
+        contypeother: '',
+        journal_name: '',
```

Comment:
You removed this (correctly) in 8e5a622 but re-added it in 0786077. It should be removed, right? Same goes for the other keys modified in those commits... might want to revert the revert.